frontend/src/components/Forms/TeamEventCreditsForm/TeamEventCreditsForm.tsx

```diff
@@ -53,7 +63,7 @@ const TeamEventCreditForm: React.FC = () => {
       const newTeamEventAttendance: TeamEventAttendance = {
         member: userInfo,
         hoursAttended: Number(hours),
-        image
+        image: new Date().toISOString()
```

Comment:
I think it might be easier to have the `image` field be the full path to the image. i.e. `eventProofs/[netid]/[timestamp]`

Reply:
I think still needs to be updated here